pkgs/development/python-modules/aioinflux/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pytz
+, aiohttp
+, ciso8601
+, pandasSupport ? true
+, pandas
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "aioinflux";
+  version = "0.9.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "gusutabopb";
+    repo = "aioinflux";
+    rev = "v${version}";
+    sha256 = "0cvzkd05i8bzh76m75s7na2gb0kh5msyyz60ajxpj2by9x6qkxmc";
+  };
+
+  propagatedBuildInputs = [ aiohttp pytz ciso8601 ]
+    ++ lib.optional pandasSupport pandas;
```

Comment:
if it's highly recommended to use pandas, then I would just add this, otherwise leave out the logic to conditionally add. Python packages are weird and it's awkward if two packages have aioinflux as a dependency, then it get's added twice.

Reply:
Pandas is one of the pretty killer features of the package, so I'll remove the condition.

Does the double-add happen when one package has pandasSupport, while the other overrides/disables it? Differing buildInputs would mean differing hashes, so different install paths. I can see how that would be non-trivial to solve.